openwisp_notifications/tasks.py

```diff
@@ -25,3 +27,14 @@ def delete_obsolete_notifications(instance_app_label, instance_model, instance_i
         | Q(target_object_id=instance_id)
     )
     Notification.objects.filter(where).delete()
+
+
+@shared_task
+def delete_expired_notifications(threshold_expiry_days):
```

Comment:
notifications do not expire, so the name is misleading. Please rename it to `delete_old_notifications`, rename `threshold_expiry_days` to simply `days`